examples/plot_simulate_somato.py

```diff
@@ -0,0 +1,114 @@
+"""
+====================
+Simulate somato data
+====================
+
+This example demonstrates how to simulate the source time
+courses obtained during median nerve stimulation in the MNE
+somatosensory dataset.
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+
+###############################################################################
+# First, we will import the packages and define the paths
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import somato
+from mne.minimum_norm import apply_inverse, make_inverse_operator
+
+data_path = somato.data_path()
+raw_fname = data_path + '/MEG/somato/sef_raw_sss.fif'
+fwd_fname = data_path + '/MEG/somato/somato-meg-oct-6-fwd.fif'
+subjects_dir = data_path + '/subjects'
+
+###############################################################################
+# Then, we get the raw data and estimage the source time course
+
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+raw.filter(1, 40)
+
+events = mne.find_events(raw, stim_channel='STI 014')
+event_id, tmin, tmax = 1, -.2, .15
+baseline = None
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, baseline=baseline,
+                    reject=dict(grad=4000e-13, eog=350e-6), preload=True)
+evoked = epochs.average()
+
+fwd = mne.read_forward_solution(fwd_fname)
+cov = mne.compute_covariance(epochs)
+inv = make_inverse_operator(epochs.info, fwd, cov)
+
+method = "MNE"
+snr = 3.
+lambda2 = 1. / snr ** 2
+stc = apply_inverse(evoked, inv, lambda2, method=method, pick_ori="normal",
+                    return_residual=False, verbose=True)
+
+pick_vertex = np.argmax(np.linalg.norm(stc.data, axis=1))
+
+plt.figure()
+plt.plot(1e3 * stc.times, stc.data[pick_vertex, :].T * 1e9, 'ro-')
+plt.xlabel('time (ms)')
+plt.ylabel('%s value (nAM)' % method)
+plt.xlim((0, 150))
+plt.axhline(0)
+plt.show()
+
+###############################################################################
+# Now, let us try to simulate the same with MNE-neuron
+
+import os.path as op
+
+import mne_neuron
+from mne_neuron import simulate_dipole, Params, Network
+
+from neuron import h
+
+mne_neuron_root = op.join(op.dirname(mne_neuron.__file__), '..')
+h.load_file("stdrun.hoc")
+
+params_fname = op.join(mne_neuron_root, 'param', 'default.json')
+params = Params(params_fname)
+
+params.update({
```

Comment:
@rythorpe I would suggest sharing the params as a "diff". That is, what do we need to change in the defaults.param or defaults.json to get the waveform. That way, it's a bit more clear to the users (and to me!) what are the canonical parameters that can be somewhat fixed and which ones are actually critical to producing the waveform.

Reply:
The waveform above is a bit different from the one produced by HNN. Here's what I get:

![image](https://user-images.githubusercontent.com/20212206/57309278-cd768980-70b5-11e9-9e65-6d13e583d904.png)

We could do that, though we would need to come to some sort of consensus regarding what we consider to be the "default" parameters. Were you thinking of just using the default.param file referenced in the HNN tutorial?

Reply:
Hmm ... the difference is probably because you have multiple trials. I made n_trials = 1 because we're still trying to figure out what's the best way to do the parallel processing with @blakecaldwell. There is an open pull request for that. There seems to be a fair amount of noise at the level of single trials. Did you try and see what it looks like for n_trials = 1? It should be the same because the random seeds etc have not changed.

Yes, currently I am just using the default.param from the HNN tutorials. But we could change this of course. Just want to change one thing at a time so we know where the differences are coming from (between softwares, models, examples etc.), if any.

Reply:
@jasmainak What are the units on the y-axis?

Reply:
should be the same as yours, that is nAm with a scaling factor of 30. Where do you get the 6000 cells from? It's not in your param file ...

Reply:
oh I see, thanks for explaining. The plots look very similar to me -- the only difference being that you have smoothing over many trials. I think the next logical step would be to get back to #40 and see what can be done to make it work in parallel.

Reply:
There is still something slightly different between our respective single-trial simulations. In particular, the initial upward deflection isn't present in your version. Here's what I get (note that the solid black line is the first single trial and the dashed line is the 100-trial average):

![N20_mne_1trial](https://user-images.githubusercontent.com/20212206/57455450-21f64200-7239-11e9-8bf5-bf66dc950844.png)